src/renderer/Generics/redesign/DynamicSettings/convertConfigToLabelSettings.tsx

```diff
@@ -64,6 +66,12 @@ const convertConfigToLabelSettings = ({
             configValuesMap,
           );
 
+          const keyDisabled =
```

Comment:
I like how this is done... that is showing the user the setting but not allowing them to change it